AirSim on Unity installation error

Hello, everyone:
When I tried to run ./build.sh in the AirSim/Unity/build.sh, the following error is raised:

./build.sh: line 14: cmake: command not found
make: *** No targets specified and no makefile found.  Stop.
cp: cannot stat 'libAirsimWrapper.so': No such file or directory

The previous steps works pretty fine. Does anyone know the solution?

[OS]
Ubuntu 18.04

--- a/Unity/build.sh
+++ b/Unity/build.sh
@@ -7,11 +7,13 @@ if [ ! -d "linux-build" ]; then
     mkdir linux-build;
 fi
 
+
 cd linux-build;
 export CC="clang-5.0"
 export CXX="clang++-5.0"
+CMAKE="$(readlink -f ../../cmake_build/bin/cmake)"
 
-cmake ../AirLibWrapper/AirsimWrapper;
+"$CMAKE" ../../cmake ../AirLibWrapper/AirsimWrapper;
 make -j`nproc`;
 if [ ! -d "../UnityDemo/Assets/Plugins" ]; then
 	mkdir ../ UnityDemo/Assets/Plugins;

